storage/engine/mvcc.go

```diff
@@ -423,57 +425,64 @@ type getValueFunc func(engine Engine, start, end proto.EncodedKey,
 // The consistent parameter specifies whether reads should ignore any
 // pending write intents and read the most recent _committed_ value
 // instead. In the event that an inconsistent read does encounter
-// intents, the intent is returned via a WriteIntentError, in addition
-// to the result.
+// an intent (currently there can only be one), it is returned via a the
+// proto.Intent slice, in addition to the result.
 func mvccGetInternal(engine Engine, key proto.Key, metaKey proto.EncodedKey, timestamp proto.Timestamp,
-	consistent bool, txn *proto.Transaction, getValue getValueFunc, buf *getBuffer) (*proto.Value, error) {
+	consistent bool, txn *proto.Transaction, getValue getValueFunc, buf *getBuffer) (*proto.Value, []proto.Intent, error) {
 	if !consistent && txn != nil {
-		return nil, util.Errorf("cannot allow inconsistent reads within a transaction")
+		return nil, nil, util.Errorf("cannot allow inconsistent reads within a transaction")
 	}
 
-	var err error
 	meta := &buf.meta
 
 	// If value is inline, return immediately; txn & timestamp are irrelevant.
 	if meta.IsInline() {
 		if err := meta.Value.Verify(key); err != nil {
-			return nil, err
+			return nil, nil, err
 		}
-		return meta.Value, nil
-	}
-	// If we're doing inconsistent reads and there's an intent, we
-	// ignore the intent by insisting that the timestamp we're reading
-	// at is a historical timestamp < the intent timestamp. However, we
-	// return a write intent error so that the intents can be resolved.
-	var wiErr error
-	if !consistent && meta.Txn != nil && !timestamp.Less(meta.Timestamp) {
```

Comment:
Likely due to intermediate changes no more present in the PR. I'm putting `!consistent` back to the front.